lib/model.ts

```diff
@@ -104,29 +104,27 @@ function generateMesh(scene: Scene, p3: boolean, rec2020: boolean): void {
   mesh.translateY(0.3)
   scene.add(mesh)
 
-  let plane = new PlaneGeometry(1, 2)
-  let planeColor = []
-  for (let i = 0; i < 2; i++) {
-    planeColor.push(0, 0, 0)
+  let plane = new PlaneGeometry(1, 1, 1, 5)
+  let planeColor = [0, 0, 0]
+  if ('array' in plane.attributes.position) {
+  let vertices = plane.attributes.position.array.length
+    for (let i = 3; i < vertices - 3; i += 3) {
+      let vertexLch: AnyLch = build(i * 2.6, 0, 0)
```

Comment:
```suggestion
      let vertexLch: AnyLch = build(i * 2.6 * L_MAX, 0, 0)
```

And then you can remove `let vertexCol = LCH ? vertexRgb.r : vertexRgb.r / 255`